pkgs/tools/backup/autorestic/default.nix

```diff
@@ -28,5 +28,6 @@ buildGoModule rec {
     license = licenses.asl20;
     maintainers = with maintainers; [ renesat ];
     platforms = platforms.linux ++ platforms.darwin;
```

Comment:
This could probably be removed this as the values that `platforms` defaults to what is currently, so this is somewhat redundant. I know it's not something touched in your changes, but since you have other cleanup changes it could be included.